bucket_sort/src/test/java/ru/nms/labs/BucketSortTest.java

```diff
@@ -0,0 +1,24 @@
+package ru.nms.labs;
+
+import org.junit.jupiter.api.Test;
+import ru.nms.labs.service.BucketSort;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class BucketSortTest {
```

Comment:
Душнота:

```suggestion

class BucketSortTest {
```

`JUnit` тестам (как и методам в них) лучше не быть `public`, чтобы не захламлять табкомплит в соседних классах